elasticsearch/client.go

```diff
@@ -25,7 +26,9 @@ func NewClient(logger *log.Logger) (*Client, error) {
 	return &Client{logger: logger}, nil
 }
 
-func (c Client) Index(i Indexable) error {
+func (c Client) Index(i Indexable) *IndexResult {
+	c.logger.Printf("Indexing %s ID %d", reflect.TypeOf(i).PkgPath(), i.GetId())
```

Comment:
Assuming you are wanting to print the type here, you can do the following instead of using reflection to get the package path:

`c.logger.Printf("Indexing %T ID %d", i, i.GetId())`

Reply:
Good point! Thanks